README.markdown

```diff
@@ -8900,6 +8900,85 @@ This API was first usable in the context of [init_by_lua*](#init_by_lua) since t
 
 This API was first enabled in the `v0.6.0` release.
 
+[Back to TOC](#nginx-api-for-lua)
+
+ngx.run_worker_thread
+----------------
+**syntax:** *ok, res1, res2, ... = ngx.run_worker_thread(threadpool, module, arg, ...)*
```

Comment:
We should just name the first `arg` to be `func_name`. It's clearer.

Reply:
I don't understand here. Could you clarify it more?

The `moudle` is the lua file to load. The `module` would be loaded as a function, and the `arg, ...` would be the argument list of that function.